buildSrc/src/test/java/org/opensearch/gradle/EmptyDirTaskTests.java

```diff
@@ -65,6 +68,7 @@ public void testCreateEmptyDir() throws Exception {
 
     public void testCreateEmptyDirNoPermissions() throws Exception {
         RandomizedTest.assumeFalse("Functionality is Unix specific", Os.isFamily(Os.FAMILY_WINDOWS));
+        RandomizedTest.assumeFalse("Functionality doesn't work in Docker", isRunningInDocker());
```

Comment:
Can we add a comment and explain the reason about why it doesn't work in docker?

Reply:
We can. And we did.